### PR TITLE
Now it's a blueprint

### PR DESCRIPTION
**Pull request by djmitche**
This merges the tooltool repo, including #3, with the relengapi-skeleton repo.  The blueprint itself doesn't actually do anything yet, but I want to get this mess in place as a basis for that more interesting pull.

### CODE REVIEW

**Comment by djmitche** [comment]
@mrrrgn @jhford @rail -- please let me know if this is OK, or if you don't care :)

**Comment by mrrrgn** [comment]
The documentation is still the Skeleton default, other than that looks good so far

File: docs/usage/relengapi-tooltool/index.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Using Skeletons
```
Comment (mrrrgn) [inline docs/usage/relengapi-tooltool/index.rst]: Should rm, or blank out perhaps?

**Comment by djmitche** [comment]
Docs tweaked

**Comment by mrrrgn** [comment]
Cool, looks like an r+ to me! :+1: :)

**Comment by Callek** [comment]
Merely skimmed, but :+1: from me as well. I'd love coverage/coveralls.io support baked in as well, but I don't consider it block worthy.

**Comment by djmitche** [comment]
Callek: feel free to make a PR.  There's not much code to cover just yet, but it's never too early!

**Pull request merged**